<launch>

	<node pkg="loam" type="scan_registration" name="ScanRegistration" />
	<node pkg="loam" type="trigger" name="Trigger" />
	<node pkg="loam" type="odom_old" name="OdomTest" />
	<node pkg="loam" type="mapping_old" name="MappingTest" />

<!--

	<node pkg="loam_back_and_forth" type="scanRegistration" name="scanRegistration" output="screen"/>
	<node pkg="loam_back_and_forth" type="transformMaintenance" name="transformMaintenance" output="screen"/>
	<node pkg="loam_back_and_forth" type="laserMapping" name="laserMapping" output="screen"/>
	<node pkg="loam_back_and_forth" type="laserOdometry" name="laserOdometry" output="screen"/>

-->
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find loam)/launch/loam.rviz"/>

</launch>
